### PR TITLE
Uninstall module CQRS command

### LINKED ISSUE #36976: Admin API - Uninstall module

### PR DESCRIPTION
**Pull request by dylanDenizonPresta**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Uninstall module CQRS command
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green
| UI Tests          | ~
| Fixed issue or discussion?     | Fixes #36976
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `branch` should be `develop` or `8.2.x`. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#branch))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Comment by ps-jarvis** [comment]
Hello @dylanDenizonPresta!

This is your first pull request on [PrestaShop](https://github.com/PrestaShop/PrestaShop/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

File: src/Adapter/Module/QueryHandler/GetModuleInfosHandler.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function handle(GetModuleInfos $query): ModuleInfos
             $module->get('name'),
             $module->get('version'),
             $module->isActive(),
-            $module->isInstalled(),
+            $module->IsInstalled(),
```
Comment (jolelievre) [inline src/Adapter/Module/QueryHandler/GetModuleInfosHandler.php]:
```suggestion
            $module->isInstalled(),
```

File: src/Core/Domain/Module/Exception/ModuleNotInstalledException.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Module\Exception;
+
+/**
+ * Is thrown when required module cannot be found
```
Comment (jolelievre) [inline src/Core/Domain/Module/Exception/ModuleNotInstalledException.php]:
```suggestion
 * Is thrown when required module is not installed
```

File: src/Core/Domain/Module/Command/UninstallModuleCommand.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+/**
+ * Uninstall module
+ */
+class UninstallModuleCommand
+{
+    /**
+     * @var string
+     */
+    private $technicalName;
```
Comment (jolelievre) [inline src/Core/Domain/Module/Command/UninstallModuleCommand.php]:
```suggestion
    private ModuleTechnicalName $technicalName;
```

File: src/Core/Domain/Module/Command/UninstallModuleCommand.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * @var bool
+     */
+    private $deleteFile;
```
Comment (jolelievre) [inline src/Core/Domain/Module/Command/UninstallModuleCommand.php]:
```suggestion
    private bool $deleteFile;
```

File: src/Core/Domain/Module/Command/UninstallModuleCommand.php [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+use PrestaShop\PrestaShop\Core\Domain\Module\ValueObject\ModuleTechnicalName;
+
+/**
+ * Uninstall module
+ */
+class UninstallModuleCommand
+{
+    /**
+     * @var string
+     */
```
Comment (jolelievre) [inline src/Core/Domain/Module/Command/UninstallModuleCommand.php]:
```suggestion
```

File: src/Core/Domain/Module/Command/UninstallModuleCommand.php [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    private $technicalName;
+
+    /**
+     * @var bool
+     */
```
Comment (jolelievre) [inline src/Core/Domain/Module/Command/UninstallModuleCommand.php]:
```suggestion
```

**Review by jolelievre — APPROVED** [review]

**Review by tleon — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**